hack/build/common.sh

```diff
@@ -45,7 +45,7 @@ ARCHITECTURE="${BUILD_ARCH:-$(uname -m)}"
 HOST_ARCHITECTURE="$(uname -m)"
 CDI_CRI="$(determine_cri_bin)"
 if [ "${CDI_CRI}" = "docker" ]; then
-   CDI_CONTAINER_BUILDCMD="docker"
+   CDI_CONTAINER_BUILDCMD=${CDI_CONTAINER_BUILDCMD:=docker}
```

Comment:
I do not think you need to get default and mutate variable `:=`, if you explicitely assignt it

`CDI_CONTAINER_BUILDCMD=${CDI_CONTAINER_BUILDCMD:-docker}` 
is probably the same as 
`${CDI_CONTAINER_BUILDCMD:=docker}` 

And this version does explicit assignment and then special default assignment, so it is redundant and surprising to read.
`CDI_CONTAINER_BUILDCMD=${CDI_CONTAINER_BUILDCMD:=docker}`

Reply:
changed it to use :-, if this is what you meant